code/modules/mob/living/living_say.dm

```diff
@@ -29,32 +29,33 @@ GLOBAL_LIST_INIT(department_radio_keys, list(
 
 
 	//kinda localization -- rastaf0
-	//same keys as above, but on russian keyboard layout. This file uses cp1251 as encoding.
+	//same keys as above, but on russian keyboard layout.
+	//updated to UTF-8, 12.02.2021
```

Comment:
```suggestion
	//same keys as above, but on russian keyboard layout.
```

We don't need comments about what code used to be.

Reply:
Done.